integreat_cms/core/management/commands/update_link_text.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import annotations
+
+import logging
+import time
+from html import unescape
+from typing import TYPE_CHECKING
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import CommandError
+from linkcheck.listeners import tasks_queue
+from linkcheck.models import Url
+from lxml.etree import ParserError
+from lxml.html import fromstring, tostring
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from django.core.management.base import CommandParser
+
+    from .regions.region import Region
+    from .users.user import User
+
+
+from ..log_command import LogCommand
+
+logger = logging.getLogger(__name__)
+
+
+def get_url(url: str) -> Region:
```

Comment:
```suggestion
def get_url(url: str) -> URL:
```
This doesn't return a Region object, right? :thinking: If it did, I think we should rename the function

Reply:
You're right 😸 Thak you!